### ISSUE TITLE
Provide more feedback during item download

### ISSUE THREAD

**Issue by EdwardBetts** [EdwardBetts/osm-wikidata#203]
Each wikidata item download triggers four wikidata queries. The user should be given updates as each query happens.

**Comment by matkoniecz** [comment]
And what user should not error - https://osm.wikidata.link/matcher/relation/224462 ends with

> wikidata query error

without indication whatever one should retry, select subarea or report a bug

**Comment by EdwardBetts** [comment]
Yes, the error needs more detail. In this case I think I might be triggering some wikidata query rate limiting, so I'm getting more timeout errors. I tried making the chunks smaller, but it still gave me a timeout.

**Comment by EdwardBetts** [comment]
This is fixed.
